src/elements/tooltip_element.js

```diff
@@ -5,6 +5,25 @@ export default class TooltipElement extends HTMLElement {
     this.shadowRoot.innerHTML = this.html
   }
 
+  connectedCallback () {
+    const savedTransform = localStorage.getItem(
+      `turbo-boost-devtools-${location.href}-${this.id}-transform`
+    )
+
+    if (savedTransform) {
+      this.style.transform = savedTransform
+    }
+  }
+
+  disconnectedCallback () {
+    if (this.id !== 'undefined' && this.id !== '') {
+      localStorage.setItem(
+        `turbo-boost-devtools-${location.href}-${this.id}-transform`,
+        this.style.transform
+      )
+    }
+  }
+
```

Comment:
Is `transform` really the only CSS property that needs to be persisted?

Reply:
it does seem so. if I find another one, I can add it...